source/usage-examples.txt

```diff
@@ -78,8 +78,8 @@ root of your project. For more information, see the
 Connection String
 ~~~~~~~~~~~~~~~~~
 
```

Comment:
Suggestion: After improving the quick start page, I think it's more chronological if this section should come before defining the env variable.